_config/debugbar.yml

```diff
@@ -51,3 +51,5 @@ SilverStripe\Admin\LeftAndMain:
 SilverStripe\Control\Controller:
   extensions:
     - LeKoala\DebugBar\Extension\ControllerExtension
+LeKoala\DebugBar\DebugBar:
+  show_namespaces: false
```

Comment:
That doesn't make sense considering all other database configs are in the same config block as well.

Either split it all out and have the config all over the place, or have it grouped in one place.